scenes/Ships/Ship_controls.gd

```diff
@@ -75,7 +75,7 @@ func set_selection():
 	for planet in planets:
 		var planetsize = planet.get_node("Selection").get_size()
 		var closestplanetsize = closest_planet.get_node("Selection").get_size()
-		if planet.global_position.distance_to(position) - planetsize < closest_planet.global_position.distance_to(position) - closestplanetsize:
+		if planet.global_position.distance_to(global_position) - planetsize < closest_planet.global_position.distance_to(global_position) - closestplanetsize:
```

Comment:
Can you explain the difference between global_position and position? It will be good for @Mokimbo and I to know the difference.

Reply:
My understanding is that position is the position of the node relative to it's parent. Global position is your actual position in the scene. I believe that by using position previously the distance calculation was being off set slightly. When I made my position changes, unrelated to the selection code, it broke the selection code because the off set was larger and thus it became clear we weren't actually selecting the closet object.